IBAnimatable/MaskDesignable.swift

```diff
@@ -191,7 +196,13 @@ public extension MaskDesignable where Self: UIView {
   
   private func maskWaveFromString(mask: String) {
     let params = retrieveMaskParameters(mask, maskName: MaskType.Wave.rawValue).componentsSeparatedByString(",")
-    if let unwrappedWidth = Float(params[1]), unwrappedOffset = Float(params[2]) where params.count == 3 {
+    
+    guard params.count == 3 else {
```

Comment:
Right, will fix it, I thought `if let unwrappedWidth = Float(params[1]), unwrappedOffset = Float(params[2]) where params.count == 3` should guard it, But it didn't, and crash the app.